tests/sles4sap/qesapdeployment/setup.pm

```diff
@@ -0,0 +1,33 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Setup and install more tools in the running jumphost image for qe-sap-deployment
```

Comment:
I don't see this module setting anything up. It's just checking that some commands are installed in the SUT/jumphost and recording their versions. Additionally, it's not used in the schedule and as such, there's not a VR for it. My suggestion would be to drop it from the PR the time being.

Code looks fine though. :)